GDDImagineGame/Assets/Team2/scripts/manager.cs

```diff
@@ -7,7 +7,9 @@ public class manager : MonoBehaviour
 
     //private var for keeping track of the time
     private float timer = 0.0f;
-
+    //public var for how long a "round" should be (time when players can choose an action)
+    public float timeToChoose = 0.0f;
```

Comment:
If you don't intend for this variable to be used in other classes, you probably want to make it private and expose it to the editor using the tag [SerializeField].

Protecc the data :)